frontend/common/src/settings/graphs.ts

```diff
@@ -210,7 +210,6 @@ export const getTimeframeByRange = (
   if (range < dayTimestamp) {
     return {
       ...usedTimeframe,
-      xAxisLabelDisplayFormat: 'MMM D HH:mm',
```

Comment:
Was this unused?

Reply:
Yes the hour and minutes supposed to show in the tooltip only, not in the axis.